pom.xml

```diff
@@ -807,9 +808,9 @@
                         <!--<include>**/server/master/register/MasterRegistryTest.java</include>-->
                         <include>**/server/master/AlertManagerTest.java</include>
                         <include>**/server/master/MasterCommandTest.java</include>
-                        <include>**/server/master/DependentTaskTest.java</include>
+                        <!--<include>**/server/master/DependentTaskTest.java</include>-->
```

Comment:
Hi~,
I have fixed the github action oom, please revert these changes.

Reply:
> I have fixed the github action oom, please revert these changes.

Ok, let me modify it